metaswitch/common/alarms.py

```diff
@@ -46,6 +46,10 @@
 sendrequest = None
 
 def issue_alarm(process, identifier):
+    # Clearwater nodes all have clearwater-infrastructure installed.
+    # It includes a command-line script that can be used to issue an alarm.
+    # Rather than refactor clearwater-infrastructure, we re-use the function
```

Comment:
I'm wholly in favour of this new passive-aggressive approach to writing comments, and intend to incorporate it into my own style.